### PR TITLE
Making sure services are in a namespace

### PR DESCRIPTION
**Pull request by AgarFu**

### CODE REVIEW

File: pkg/kubernetes/kube.go [hunk 1]
```diff
@@ -551,18 +551,28 @@ func (kc *Client) DeleteAllTestResources() error {
 }
 
 /*
-DeploymentInNamespace check if deployment in the related namespace
+ResourceInNamespace check if (deployment|servuce) in the related namespace
```
Comment (zhatt) [inline pkg/kubernetes/kube.go:554]:
```suggestion
ResourceInNamespace check if (deployment|service) in the related namespace
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Default Go linker flags.
+GO_LDFLAGS := -ldflags="-s -w"
+
+# Step Functions
+BINARY := kubedog
+
+.PHONY: all
+all: clean test build style vet lint
```
Comment (zhatt) [inline Makefile:8]: clean should not be part of the default target.

**Review by zhatt — APPROVED** [review]
Looks good thanks.

**Review by garomonegro — APPROVED** [review]
LGTM. Thanks! @AgarFu

**Review by zhatt — APPROVED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,4 +1,6 @@
 .vscode
 .DS_Store
 *.test
-test/templates/generated_*
\ No newline at end of file
+test/templates/generated_*
+kubedog
+coverage.out
```
Comment (vgunapati) [inline .gitignore:6]:
```suggestion
coverage.out

# ignore edit files
.*~
.*.sw?
```

adding this to just to retrigger a build

**Pull request merged**